gwpy/tests/test_io.py

```diff
@@ -35,8 +35,12 @@
                      gwf as io_gwf,
                      kerberos as io_kerberos,
                      nds2 as io_nds2,
+<<<<<<< HEAD
```

Comment:
@areeda, please resolve this conflict, this is the primary reason for the travis-ci failures.